plugin/src/App/StatusPages/Settings/init.lua

```diff
@@ -111,20 +112,28 @@ function SettingsPage:render()
 				layoutOrder = 3,
 			}),
 
+			OpenScriptsExternally = e(Setting, {
+				id = "openScriptsExternally",
+				name = "Open Scripts Externally",
+				description = "EXPERIMENTAL! Attempt to open scripts in an external editor",
+				transparency = self.props.transparency,
+				layoutOrder = 4,
+			}),
+
 			TwoWaySync = e(Setting, {
 				id = "twoWaySync",
 				name = "Two-Way Sync",
 				description = "EXPERIMENTAL! Editing files in Studio will sync them into the filesystem",
 				transparency = self.props.transparency,
-				layoutOrder = 4,
+				layoutOrder = 5,
 			}),
 
 			LogLevel = e(Setting, {
 				id = "logLevel",
 				name = "Log Level",
 				description = "Plugin output verbosity level",
 				transparency = self.props.transparency,
-				layoutOrder = 5,
+				layoutOrder = 100,
```

Comment:
Why is the layout order for this and the typechecking setting so high? I'm not intimately familiar with the plugin, so forgive me if this is an obvious question.

Reply:
These are dev settings, not really used by consumers, so they're set really high so that they'll always be last in the list and out of the way of users without us having to manually update their layoutorder by one every time we add a setting.